UIKit/Tasks/MenuViewController.swift

```diff
@@ -5,9 +5,189 @@ import UIKit
+        image.frame = CGRect(x: 270, y: 505, width: 70, height: 70)
+        return image
+    }()
+
+    private let coffeButton: UIButton = {
+        let button = UIButton()
+        button.setTitleColor(.black, for: .normal)
+        button.setTitle("Кофий", for: .normal)
+        button.titleLabel?.font = UIFont(name: "Verdana-BoldItalic", size: 16)
+        button.frame = CGRect(x: 5, y: 710, width: 180, height: 19)
+        button.addTarget(self, action: #selector(pushToCheque), for: .touchUpInside)
+        return button
+    }()
+
+    private let hotButton: UIButton = {
+        let button = UIButton()
+        button.setTitleColor(.black, for: .normal)
+        button.setTitle("Горячiя напитки", for: .normal)
+        button.titleLabel?.font = UIFont(name: "Verdana-BoldItalic", size: 16)
+        button.frame = CGRect(x: 42, y: 620, width: 180, height: 19)
+        return button
+    }()
+
+    private let ptiButton: UIButton = {
+        let button = UIButton()
+        button.setTitleColor(.black, for: .normal)
+        button.setTitle("Пти пате аля «РюсЪ»", for: .normal)
+        button.titleLabel?.font = UIFont(name: "Verdana-BoldItalic", size: 16)
+        button.frame = CGRect(x: 45, y: 530, width: 220, height: 19)
+        return button
+    }()
+
+    private let thirdVarMenuImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "rect")
+        image.frame = CGRect(x: 21, y: 680, width: 335, height: 80)
+        return image
+    }()
+
+    private let secondVarMenuImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "rect")
+        image.frame = CGRect(x: 20, y: 590, width: 335, height: 80)
+        return image
+    }()
+
+    private let variantsMenuImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "variants")
+        image.frame = CGRect(x: 20, y: 500, width: 335, height: 80)
+        return image
+    }()
+
+    private let menuImage: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "menu")
+        image.frame = CGRect(x: 120, y: 410, width: 125, height: 80)
+        return image
+    }()
+
+    private let pointViewLabel: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "po")
+        image.frame = CGRect(x: 320, y: 350, width: 20, height: 29)
+        return image
+    }()
+
+    private let textLabelPointView: UILabel = {
+        let label = UILabel()
+        label.text = "Разрѣшите доступъ къ ​геолокаціи для поиска ближайшей кофейни"
+        label.font = UIFont(name: "Verdana", size: 12)
+        label.textColor = #colorLiteral(red: 0.6120845675, green: 0.6320224404, blue: 0.6574426293, alpha: 1)
+        label.numberOfLines = 0
+        label.frame = CGRect(x: 30, y: 360, width: 260, height: 30)
+        return label
+    }()
+
+    private let labelPointView: UILabel = {
```

Comment:
```suggestion
    private let pointViewLabel: UILabel = {
```
не совсем понимаю что за лейбл точка